backend/internal/service/custom_claim_service.go

```diff
@@ -25,6 +25,7 @@ func isReservedClaim(key string) bool {
 		"name",
 		"email",
 		"preferred_username",
+		"display_name",
```

Comment:
Could this be a breaking change if people have defined a claim called `display_name`?